vertx/src/main/java/com/microsoft/gctoolkit/vertx/jvm/JvmConfigurationImpl.java

```diff
@@ -2,62 +2,102 @@
 // Licensed under the MIT License.
 package com.microsoft.gctoolkit.vertx.jvm;
 
-import com.microsoft.gctoolkit.parser.jvm.JVMConfiguration;
+import com.microsoft.gctoolkit.jvm.Diary;
+import com.microsoft.gctoolkit.time.DateTimeStamp;
 
 /* package scope */ class JvmConfigurationImpl implements com.microsoft.gctoolkit.jvm.JvmConfiguration {
 
-    private final JVMConfiguration jvmConfiguration;
+    private final Diary diary;
     /* package scope */
-    JvmConfigurationImpl(JVMConfiguration jvmConfiguration) {
-        this.jvmConfiguration = jvmConfiguration;
+    JvmConfigurationImpl(Diary diary) {
```

Comment:
Do we need this impl class? Since this class is essentially a wrapper around Diary, can JvmConfiguration be changed from an interface to a class?